ansible_rulebook/cli.py

```diff
@@ -38,6 +38,7 @@
     job_template_runner,
 )
 
+DEFAULT_GC_AFTER = 1000
```

Comment:
This value should be defined in `ansible_rulebook.conf` to avoid duplication and side effects, because the same value is hardcoded as default in the `_Settings` object,